src/modal/modal-stack.ts

```diff
@@ -132,7 +131,9 @@ export class NgbModalStack {
   }
 
   private _createFromTemplateRef(content: TemplateRef<any>, context: NgbActiveModal): ContentRef {
-    const viewRef = content.createEmbeddedView(context);
+    const $context =
+        {$implicit: context, close(result) { context.close(result); }, dismiss(reason) { context.dismiss(reason); }};
```

Comment:
Could you please rename:
* `context` => `activeModal`
* `$context` => `tplRefContext` / `context`

Reply:
Done